doc/users/next_whats_new/2020-05-26-cl-subplot.rst

```diff
@@ -0,0 +1,17 @@
+Constrained layout now supports subplot and subplot2grid
```

Comment:
Or, "Subplot and subplot2grid can now work with constrained layout".  (I used "can" because they work only if called appropriately.)